docs/hooks/hook_civicrm_permission.md

```diff
@@ -29,7 +29,7 @@ to grant it to users.
     -   Simple associative array in the format "permission string =>
         label".  Compatible with CiviCRM 4.3+.
 
-            $prefix = ts('CiviCRM Grant Program') . ': '; // name of extension or module
+            $prefix = ts('CiviCRM Grant Program:') . ' ';
```

Comment:
@mlutfy just looking at this a couple of thoughts

1. should this be E::ts or similar and
2. the previous code appears to be at least consistant with core https://github.com/civicrm/civicrm-core/blob/master/CRM/Core/Permission.php#L687 not sure if that is correct or not